%!TEX root = NoveltyTagm_rev1_clean.tex
%*******************************************************************************
%****************************** Supplement *********************************
%*******************************************************************************
\section*{Appendix}
\subsection*{mESC chromatin enrichment validation}\label{supplement:mesc}
For the mESC dataset, Novelty TAGM reveals $8$ new putative phenotypes. Novelty TAGM recovers the masked annotations with phenotype $2$ having the enriched terms associated with chromatin, such as \textit{chromatin} and \textit{chromosome} $(p < 10^{-80})$. Phenotype 3 corresponds to a separate nuclear substructure with enrichment for the terms \textit{nucleolus} $(p < 10^{-60})$ and \textit{nuclear body} $(p < 10^{-30})$. Thus, in the mESC dataset Novelty TAGM confirms the chromatin enrichment preparation designed to separate chromatin and non-chromatin associated nuclear proteins \citep{Mulvey:2017}. In addition, phenotype 4 demonstrates enrichment for the ribosome annotation $(p < 10^{-35})$. Phenotype 1 is enriched for \textit{centrosome} and \textit{microtubule} annotations $(p < 10^{-15})$, though observing the PSM in Fig\ref{figure:mouse} we can see there is much uncertainty in this phenotype. This uncertainty quantification can then be used as a basis for justifying additional expert annotation.

\begin{figure}[h]
	\begin{subfigure}[t]{0.5\textwidth}
	\centering
	\includegraphics[height=2.5in]{pcaMousehl.pdf}
	\caption{}
\end{subfigure}
\begin{subfigure}[t]{0.5\textwidth}
	\centering
	\includegraphics[height=2.5in]{heatmapMousehl.jpeg}
	\caption{}
\end{subfigure}
\caption{(a) PCA plot of the \textit{hyper}LOPIT mESC dataset. Points are scaled according to the discovery probability. (b) Heatmaps of the posterior similarity matrix derived from  mESC data demonstrating the uncertainty in the clustering structure of the data. We have only plotted the proteins which have greater than 0.99 probability of belonging to a new phenotype and probability of being an outlier less than $0.95$ for the mESC dataset to reduce the number of visualised proteins.}
\label{figure:mouse}
\end{figure}
\subsection*{Uncovering additional annotations in fibroblast cells}\label{supplement:fibroblast}
\subsubsection*{HCMV-infected fibroblast cells} 
We apply Novelty TAGM to the dataset corresponding to the HCMV-infected fibroblast cells 24 hours post infection (hpi) \citep{Jean_Beltran:2016}, and discover $9$ putative additional phenotypes (demonstrated in Fig\ref{figure:Beltran}). Phenotype $2$ contains a singleton protein and phenotypes $4,6,7,8$ and $9$ are not significantly enriched for any annotations. However, phenotype $3$ is enriched for the \textit{mitochondrial membrane} and  \textit{mitochondrial envelope} annotations $(p < 10^{-4})$; this is an addition to the already annotated mitochondrial class, indicating sub-mitochondrial resolution. Phenotype $1$ is a mixed ribosomal/nuclear cluster with enrichment for \textit{nucleoplasm} $(p < 10^{-5})$ and the \textit{small ribosomal subunit} $(p < 10^{-4})$, which is distinct from phenotype $5$ which is enriched for the \textit{large ribosomal subunit} $(p < 10^{-10})$. This demonstrates unbiased separation of the two ribosomal subunits, which was overlooked in the original analysis \citep{Jean_Beltran:2016}.

\subsubsection*{Fibroblast cells without infection} 
Novelty TAGM reveals $7$ putative phenotypes in the control fibroblast dataset \citep{Jean_Beltran:2016}. Phenotypes $2,4,5,6$ and $9$ have no significantly enriched Gene Ontology terms (threshold $p = 0.01$). However, we observe that phenotype $3$ is enriched with the \textit{large ribosomal subunit} with significance at level $p < 10^{-7} $. Phenotype $1$ represents a mixed \textit{peroxisome} $(p < 10^{-2})$ and \textit{mitochondrion} cluster $(p < 10 ^{-2})$, an unsurprising result since these organelles possess similar biochemical properties and therefore similar profiles during density gradient centrifugation-based fractionation \citep{DC:2018, Dealtry:1992}. The differing number of confidently identified and biologically relevant phenotypes discovered between the two fibroblast datasets could be down to the differing levels of structure between the two datasets. Indeed, it is evident from Fig\ref{figure:Beltran} that we see differing levels of clustering structure in these datasets.
\begin{figure}
	\begin{subfigure}[t]{0.5\textwidth}
		\centering
		\includegraphics[height=2.5in]{pcaBeltranH.pdf}
		\caption{}
	\end{subfigure}	
	\begin{subfigure}[t]{0.5\textwidth}
		\centering
		\includegraphics[height=2.5in]{heatmapHCMV.pdf}
		\caption{}
	\end{subfigure}
	~
	\begin{subfigure}[t]{0.5\textwidth}
		\centering
		\includegraphics[height=2.5in]{pcaMOCK.pdf}
		\caption{}
	\end{subfigure}
	\begin{subfigure}[t]{0.5\textwidth}
		\centering
		\includegraphics[height=2.5in]{heatmapMOCK.pdf}
		\caption{}
	\end{subfigure}
	\caption{(a, c) PCA plots of the HCMV-infected fibroblast data 24 hpi and the mock fibroblast data 24 hpi. The points are coloured according to the organelle or proposed new phenotype and are scaled according to the discovery probability. (b, d) Heatmaps of the posterior similarity matrix derived from the infected fibroblast data and mock fibroblast data demonstrating the uncertainty in the clustering structure of the data. We have only plotted the proteins which have greater than 0.99 probability of belonging to a new phenotype and probability of being an outlier less than 0.95.}
	\label{figure:Beltran}
\end{figure}
\subsection*{Additional organellar map datasets}\label{supplement:DOM}
\subsubsection*{Mouse primary neurons}
The mouse primary neuron dataset reveals $10$ phenotypes after we apply Novelty TAGM. However, $8$ of these phenotypes have no enriched GO annotations. This is likely a manifestation of the dispersed nature of this dataset, where the variability is generated by technical artefacts rather than biological signal. Despite this, Novelty TAGM is able to detect two relevant phenotypes: the first phenotype is enriched for \textit{nucleolus} $(p < 0.01)$; the second for \textit{chromosome} $(p < 0.01)$. This suggests additional annotations for this dataset.
\subsubsection*{HeLa cells (Hirst et. al 2018)}
The HeLa dataset of \cite{Hirst:2018}, which we refer to as HeLa Hirst, reveals $7$ phenotypes with at least 1 protein with discovery probability greater than $0.95$. However, three of these phenotypes represent singleton proteins. Phenotype $1$ reveals mixed cytosol/ribosomal annotations with the terms \textit{cytosolic ribosome} $(p < 10^{-30})$ and \textit{cytosolic part} $(p < 10^{-25})$ significantly over-represented. There are no further phenotypes with enriched annotations $(\text{threshold } p = 0.01)$, except phenotype $2$ which represents a mixed extracellular structure/cytosol cluster. For example, the terms \textit{extracellular organelle} $(p <10^{-13})$ and \textit{cytosol} $(p < 10^{-10})$ are over-represented.
\begin{figure}
	\begin{subfigure}[t]{0.5\textwidth}
	\centering
	\includegraphics[height=2.5in]{pcaItzhak2017.pdf}
	\caption{}
\end{subfigure}
\begin{subfigure}[t]{0.5\textwidth}
	\centering
	\includegraphics[height=2.5in]{heatmapItzhak2017.jpeg}
	\caption{}
\end{subfigure}
~
\begin{subfigure}[t]{0.5\textwidth}
	\centering
	\includegraphics[height=2.5in]{pcaHirst.pdf}
	\caption{}
\end{subfigure}
\begin{subfigure}[t]{0.5\textwidth}
	\centering
	\includegraphics[height=2.5in]{heatmapHirst.jpeg}
	\caption{}
\end{subfigure}
	\caption{(a),(c) PCA plots of the mouse primary neuron data and HeLa Hirst data. The pointers are scaled according to their discovery probability. (b),(d) Heatmaps of the mouse neuron data and HeLa Hirst data. Only the proteins whose discovery probability is greater than $0.99$ and outlier probability less than $0.95$ ($10^{-2}$ for the mouse primary neuron dataset to reduce the number of visualised proteins) are shown. The heatmaps demonstrate the uncertainty in the clustering structure present in the data.}
\end{figure}




\subsection*{Handling label switching}
Bayesian inference in mixture models suffers from an identifiability issue known as \textit{label switching} - a phenomenon where the allocation labels can flip between runs of the algorithm \citep{Richardson::1997, Stephens:2000}. This occurs because of the symmetry of the likelihood function under permutations of these labels. We note that this only occurs in unsupervised or semi-supervised mixture models. This makes inference of the parameters in mixture models challenging. In our setting the labels for the known components do not switch, but for the new phenotypes label switching must occur. One standard approach to circumvent this issue is to form the so-called \textit{posterior similarity matrix} (PSM) \citep{fritsch::2009}. The PSM is an $N\times N$ matrix where the $(i,j)^{th}$ entry is the posterior probability that protein $i$ and protein $j$ reside in the same component. More precisely, if we let $S$ denote the PSM and $T$ denote the number of Monte-Carlo iterations then 
\begin{equation}
S_{ij} = P(z_i = z_j| X, \boldsymbol{\theta}, \boldsymbol{\pi} , \kappa, \epsilon, {\bf M}, V) \approx \frac{1}{T} \sum_{t = 1}^{T} \mathds{I}(z^{(t)}_i = z^{(t)}_j),
\end{equation}
where $\mathds{I}$ denotes the indicator function. The PSM is clearly invariant to label switching and so avoids the issues arising from the \textit{label switching} problem. 
\subsection*{Summarising posterior similarity matrices}
To summarise the PSMs, we take the approach proposed by \cite{fritsch::2009}. They proposed the adjusted Rand index (AR) \citep{rand::1971, Hubert::1985}, a measure of cluster similarity, as a utility function and then we wish to find the allocation vector $\hat{z}$ that maximises the expected adjusted Rand index with respect to the true clustering $z$. Formally, we write
\begin{equation}
\hat{z} = \arg \max_{z^*} E[AR(z^*, z)|X, \boldsymbol{\theta}, \boldsymbol{\pi} , \kappa, \epsilon, {\bf M}, V],
\end{equation}
which is known as the Posterior Expected Adjusted Rand index (PEAR). One obvious pitfall is that this quantity depends on the unknown true clustering $z$. However, this can be approximated from the MCMC samples:
\begin{equation}
PEAR \approx \frac{1}{T} \sum_{t = 1}^{T}AR(z^*, z^{(t)}).
\end{equation}
The space of all possible clustering over which to maximise is infeasibly large to explore. Thus we take an approach taken in \cite{fritsch::2009} to propose candidate clusterings over which to maximise. Using hierarchical clustering with distance $1 - S_{ij}$, the PEAR criterion is computed for clusterings at every level of the hierarchy. The optimal clustering $\hat{z}$ is the allocation vector which maximises the PEAR.
\subsection*{Details of MCMC}
The MCMC algorithm used in \cite{Crook:2018} is insufficient to handle inference of unknown phenotypes. As in \cite{Crook:2018}, a collapsed Gibbs sampler approach is used, but a number of modifications are made. Firstly, to accelerate convergence of the algorithm half the proteins are initial allocated randomly amongst the new phenotypes. Secondly, the parameters for the new phenotypes are proposed from the prior. Throughout the same default prior choices are used as in \cite{Crook:2018}.

\subsection*{Further details of endosomal proteins}
For completeness, this appendix provides additional details and important literature on the proteins discussed in section \nameref{section:endo}.

First, P20339 (Rab5a) and P61020 (Rab5b) are two of the three isoforms of Rab5, a small GTPase which belongs to the Ras protein superfamily and is considered a master organiser of the endocytic system. Rab5a and Rab5b share a high level of amino acid sequence identity (approximately 85\%) and are ubiquitously expressed in the mouse and human. Independently, these isoforms act as key regulators of clathrin-mediated endocytosis and early endosome dynamics by controlling the following processes \textit{in vivo} and \textit{in vitro}: (a) clathrin-coated vesicle formation at the cell surface; (b) endocytosed vesicle transport from the plasma membrane towards, and fusion with, early endosomes; (c) early endosome biogenesis and maintenance; (d) molecular motor-driven, microtubule-dependent early endosome motility along the endocytic route; (e) early endosome docking/tethering and homotypic fusion, and (f) Rab conversion and early-to-late endosome maturation
\citep{Simonsen::1998, Zerial::2001, Rink::2005, Chen::2014, Gautreau::2014, Law::2017}.

Rab5a and Rab5b play crucial roles in the internalisation and recycling/degradation of cell surface receptors such as EGFR (epidermal growth factor receptor), TfR (transferrin receptor) and several GPCRs (G-protein-coupled receptors) and integrins as well as peripheral plasma membrane-associated signalling molecules, thereby regulating important intracellular signal transduction pathways \citep{Trischler::1999, Chen::2009, Bastin::2013, Liu::2015}. We observe a mixed steady-state potential localisation between the endosome and PM for both Rab5a and Rab5b (Fig \ref{figure:u2os}D). According to previously published information, both Rab5a and Rab5b are mainly localised to (and considered well-established constituents of) the early endosome compartment but have also been detected on the PM and clathrin-coated vesicles, in support of our results \citep{Simonsen::1998, Woodman::2000, Mendoza::2013}. Moreover, according to the HPA Cell Atlas, Rab5b resides in the vesicles (which, in this context, include the endosomes, lysosomes, peroxisomes and lipid droplets). There is no information regarding the sub-cellular location of Rab5a in this database.  

Second, Q92738 (RN-tre) is a GTPase-activating protein (GAP) which controls the activity of several Rab GTPases. RN-tre is a major Rab5 (see above) regulator and therefore a key player in the organisation and dynamics of the endocytic pathway \citep{Lanzetti::2000, Gautreau::2014}. This protein modulates the internalisation of and signal transduction mediated by cell surface receptors such as EGFR, TfR and $\beta 1$ integrins \citep{Lanzetti::2000, Martinu::2002, Palamidessi::2013, De::2015}. It also controls early endosome-to-Golgi retrograde transport and Golgi membrane organisation \citep{Haas::2007}. We observe a steady-state snapshot of the sub-cellular distribution of RN-tre with potential localisation to the endosome and PM (Fig \ref{figure:u2os}D). In line with these results, RN-tre has been shown to reside in Rab5-positive early endosomes at steady state, but has also been detected at the PM and focal adhesions \citep{Lanzetti::2000,Martinu::2002, Palamidessi::2013,Gautreau::2014, De::2015}. There is no information concerning the sub-cellular localisation of RN-tre in the HPA Cell Atlas database. 

Third, Q96L93 (KIF16B) is a plus end-directed molecular motor which belongs to the kinesin-3 protein family. This kinesin regulates early endosome motility along microtubules and is required for the establishment of the steady-state sub-cellular distribution of early endosomes as well as the balance between PM recycling and lysosome degradation of signal transducing cell surface receptors including EGFR and TfR \citep{Hoepfner:2005, Carlucci::2010}. In neuronal cells, KIF16B plays an important role in the establishment of somatodendritic early endosome localisation and in the trafficking of AMPA and NGF receptors \citep{Farkhondeh::2015}. In epithelial cells, this protein controls the transcytosis of TfR from juxtanuclear recycling endosomes to apical recycling endosomes \citep{Bay::2013}. KIF16B is also involved in tubular endosome biogenesis and fission by regulating early endosome fusion \citep{Skjeldal::2012}. Lastly, this kinesin has been shown to mediate biosynthetic Golgi-to-endosome transport of FGFR (fibroblast growth factor receptor)-carrying vesicles and thereby control FGFR cell surface presentation and signalling during in vivo mouse embryogenesis \citep{Ueno::2011}. Our results indicate a mixed localisation to the endosome and PM for KIF16B (Fig \ref{figure:u2os}D). In line with our observations, it has been reported that this protein is associated with early endosome membranes at steady state in mouse and human cells \citep{Hoepfner:2005, Farkhondeh::2015}. Additionally, it has been demonstrated that KIF16B co-localises with, and its spatial distribution and activity is regulated by, the small GTPase Rab5, whose isoforms Rab5a and Rab5b we also identified as potentially localised to the endosome and PM in the U-2 OS \textit{hyper}LOPIT dataset (see above), on early endosomes \citep{Hoepfner:2005,Skjeldal::2012}. Taking the above into account, a mixed distribution between the endosome and PM is reflective of the molecular function of KIF16B. However, the HPA Cell Atlas database classifies KIF16B as a component of the mitochondria (Fig \ref{figure:u2os}B), contradicting our findings as well as previously published information regarding the sub-cellular localisation and biological role of this protein. We speculate that this disagreement arises from the uncertainty associated with the specificity of the chosen antibody \citep{Thul:2017}. Indeed, the reliability of the mitochondrial annotation for KIF16B is classified as "uncertain" in this database.

Fourth, Q8NHG8 (ZNRF2) is an E3 ubiquitin ligase which has been shown to regulate mTOR signalling as well as lysosomal acidity and homeostasis in mouse and human cells \citep{Hoxhaj:2016}. This protein has been found to control the sub-cellular localisation and biological function of mTORC1, the V-ATPase and the Na+/K+-ATPase $\alpha1$ \citep{Hoxhaj::2012, Hoxhaj:2016}. ZNRF2 is membrane-associated but can be released into the cytosol upon phosphorylation by various kinases \citep{Hoxhaj:2016}. We observe a mixed steady-state distribution between the endosome and PM for this protein (Fig \ref{figure:u2os}D). In support of this result, we find that ZNRF2 has been detected on the endosomes, lysosomes, Golgi apparatus and PM according to the literature \citep{Araki:2003, Hoxhaj:2016}. There is no information in regard to the sub-cellular location of ZNRF2 in the HPA Cell Atlas database. 

Fifth, O15498 (Ykt6) is a SNARE (soluble N-ethylmaleimide-sensitive factor attachment protein receptor) protein which is conserved from yeast to humans. This protein regulates a wide variety of intracellular trafficking and membrane tethering and fusion processes including ER-to-Golgi vesicular transport, intra-Golgi traffic, retrograde Golgi-to-ER transport, retrograde endosome-to-TGN (trans-Golgi network) trafficking, homotypic fusion of ER membranes, Golgi-to-PM transport and exosome/secretory vesicle-PM fusion, Golgi-to-vacuole traffic (in yeast), homotypic vacuole fusion (in yeast), autophagosome formation and autophagosome-lysosome fusion \citep{Dilcher::2001, Tai:2004, Takats::2018, Matsui::2018, Linnemannstons::2018, Yong::2019}. Ykt6 lacks a transmembrane domain and is able to cycle between intracellular membranes and the cytosol in a palmitoylation- and farnesylation-dependent manner \citep{Fukasawa::2004,Meiringer::2008}. The membrane-associated form of Ykt6 has been detected on the PM, ER, Golgi apparatus, endosomes, lysosomes, vacuoles (in yeast), and autophagosomes as part of various SNARE complexes \citep{Dilcher::2001, Tai:2004,Fukasawa::2004,Meiringer::2008, Takats::2018, Matsui::2018, Linnemannstons::2018, Yong::2019}. In line with this information, our results show a mixed sub-cellular distribution for Ykt6 with potential localisation to the endosome and cytosol (Fig \ref{figure:u2os}D). The cytosolic localisation for Ykt6 is also supported by the HPA Cell Atlas annotation corresponding to this protein (Fig \ref{figure:u2os}B), further reinforcing our findings.

Sixth, Q9NZN3 (EHD3) is another important regulator of endocytic trafficking and recycling. This protein promotes the biogenesis and stabilisation of tubular recycling endosomes by inducing early endosome membrane bending and tubulation \citep{Bahl::2016,Henmi::2016}. Additionally, EHD3 is essential for early endosome-to-recycling endosome transport, retrograde early endosome-to-Golgi traffic, Golgi apparatus morphology maintenance, and recycling endosome-to-PM transport \citep{Naslavsky::2006,Naslavsky::2009,George::2007,Cabasso::2015, Henmi::2016}. It plays an important role in the recycling of cell surface receptors and the biosynthetic transport of lysosome proteins \citep{Naslavsky::2006,Naslavsky::2009,George::2007,Cabasso::2015}. We observe a mixed steady-state potential localisation to the endosome and PM for EHD3 (Fig \ref{figure:u2os}D). Our results are in agreement with previously published studies which have reported that EHD3 is resident in the early endosomes and recycling endosomes at steady state \citep{Naslavsky::2006,Naslavsky::2009,George::2007,Cabasso::2015}, and our PM localisation-related observation is supported by the HPA Cell Atlas-derived annotation for this protein (Fig \ref{figure:u2os}B).

Our findings provide insights on the dynamic sub-cellular distribution of proteins which play important roles in development, physiology and disease. For example, Rab5/Rab5a has been identified as a master regulator of cancer cell migration, tumour invasion and dissemination programs \textit{in vitro} and \textit{in vivo}. It has been demonstrated that Rab5/Rab5a expression is dysregulated in many invasive human cancers, Rab5/Rab5a is overexpressed in metastatic foci compared to the matched primary tumours, and Rab5/Rab5a activity critically promotes the acquisition of invasive properties by poorly invasive tumour cell types \citep{Torres::2010,Liu::2011,Liu::2015, Mendoza::2013,Frittoli::2014,Diaz::2014, Saitoh::2017}. Several publications have reported that elevated Rab5/Rab5a expression correlates with, and is predictive of, increased local invasiveness and metastatic potential, as well as poor patient prognosis in a variety of human cancer types \citep{Yu::1999,Fukui::2007,Zhao::2010,Yang::2011,Mendoza::2013, Frittoli::2014,Diaz::2014, Igarashi::2017}. Due to its established role in cancer progression and metastasis, Rab5/Rab5a is considered a fundamental cancer-associated protein and a potential diagnostic marker or therapeutic target \citep{Frittoli::2014, Igarashi::2017}. Recently, Rab5 was identified as a promising therapeutic target for colorectal cancer, as inhibition of Rab5 (and Rab7) activity led to elimination of colorectal cancer stem cells and disruption of colorectal cancer foci \citep{Takeda::2019}. Moreover, individual ablation of Rab5a but also Rab5b was shown to impair the invasion and dissemination ability of different cancer cell types \citep{Frittoli::2014}. In addition to its important role in cancer, there is some evidence suggesting that Rab5a might also be involved in the early pathogenesis of Alzheimer's disease \citep{Cataldo::1997,Cataldo::2000, Rosenfeld::2001}. Lastly, the Rab5 machinery has also been identified as an important factor in several bacterial, parasitic and viral infections. Bacterial pathogens such as \textit{Mycobacterium tuberculosis}, \textit{Listeria monocytogenes}, \textit{Tropheryma whipplei} and \textit{Salmonella typhimurium} \citep{Madan::2008}, as well as parasites such as \textit{Leishmania donovani} have evolved specific subversion mechanisms with which they are able to control the intracellular distribution and/or activity of Rab5 and its effectors as a way to avoid neutralisation by the immune system or facilitate invasion \citep{Verma::2017}. \textit{L. donovani} specifically controls the expression and function of the Rab5a isoform in this context \citep{Verma::2017}. Additionally, Rab5 was shown to participate in adenovirus endocytosis \citep{Rauma::1999}, both Rab5a and Rab5b were found to play functional roles in web formation and viral genome replication during HCV (hepatitis C virus) infection \citep{Stone::2007}, and Rab5a was identified as a crucial target of HBV (hepatitis B virus) during HBV-related hepatocellular carcinoma  pathogenesis \citep{Sheng::2014}.

Apart from Rab5a and Rab5b, the other proteins also possess demonstrated roles in development and disease. RN-tre is overexpressed in a subset of aggressive basal-like breast cancers, where high levels of this protein prevent the endocytosis and recycling of EGFR, leading to Akt overstimulation. In turn, Akt activity stabilises the glucose transporter GLUT1 at the cell membrane, resulting in an increase in glycolysis and cancer cell proliferation. RN-tre has been proposed as a potential therapeutic target for these types of breast cancer \citep{Avanzato::2018}. This protein also plays a functional role in infection, as it was shown to regulate the uptake and intracellular trafficking of Shiga toxins \citep{Fuchs::2007}. Furthermore, it has been reported that KIF16B is essential for early post-implantation mouse embryo development, as Kif16b-knockout animals display peri-implantation embryonic lethality \citep{Ueno::2011}. In addition, recent studies have shown that ZNRF2 is overexpressed in human non-small cell lung cancer, osteosarcoma and papillary thyroid cancer, and that high levels of this protein are correlated with disease progression and poor patient prognosis in these cases \citep{Zhang::2016,Xiao::2017,Cui::2019}. Moreover, Ykt6 was found to be necessary for glycosome biogenesis and function in the kinetoplastid parasite \textit{Trypanosoma brucei}, which causes African sleeping sickness, with Ykt6 ablation significantly reducing the viability of the parasite in both its pro-cyclic and bloodstream forms \citep{Banerjee::2017}. Finally, EHD3 has been identified as an essential factor for heart physiology \citep{Curran::2014}.
\subsection*{Summary of convergence diagnostics}
We provide a summary of convergence diagnostics using parallel chains analysis \citep{Gelman:1992}. We compute the number of proteins allocated to the outlier component at each iteration of the Markov-chain and monitor this quantity for convergence. The $\hat{R}$ statistic between parallel chains in then computed and reported in the table below. A value of $\hat{R} < 1.2$ indicates convergence.
\begin{table}[h]
	\centering
	\footnotesize
	\begin{tabular}{ |p{5cm}|p{3cm}|p{2cm}|p{4cm}|  }
		\hline
		\multicolumn{4}{|c|}{Convergence diagnostics for MCMC} \\
		\hline
		\vfill\hfil Dataset & \vfill\hfil Protocol & \vfill\hfil$\hat{R}$ & \vfill\hfil Upper confidence Interval \vfill\hfil $\hat{R}$   \\
		\hline
		\hline
		mESC  & \textit{hyper}LOPIT&\vfill\hfil 1.03 & \vfill\hfil 1.15 \\
		\hline
		U-2 OS &\textit{hyper}LOPIT & \vfill\hfil 1.00  & \vfill\hfil 1.00\\
		\hline
		U-2 OS &LOPIT-DC  & \vfill\hfil 1.02&\vfill\hfil 1.06\\
		\hline
		\textit{S. cerevisiae} &\textit{hyper}LOPIT & \vfill\hfil 1.00 & \vfill\hfil 1.01\\
		\hline
		HCMV-infected fibroblast& Spatio-Temporal Proteomics &  \vfill\hfil 1.01 & \vfill\hfil 1.02 \\
		\hline
		HCMV mock fibroblast& Spatio-Temporal Proteomics &  \vfill\hfil 1.03 & \vfill\hfil 1.08 \\
		\hline
		HeLa \citep{Itzhak:2016}& DOM &  \vfill\hfil 1.07 & \vfill\hfil 1.21 \\
		\hline
		Mouse primary neurons & DOM &  \vfill\hfil 1.04 & \vfill\hfil 1.13 \\
		\hline
		HeLa \citep{Hirst:2018} & DOM &\vfill\hfil 1.02 & \vfill\hfil 1.06 \\
		\hline
		HEK-293 &LOPIT &  \vfill\hfil 1.00 & \vfill\hfil 1.01 \\
		\hline
		\hline
	\end{tabular}
	\caption{A table reporting convergence diagnostics for MCMC analysis}
	\label{table:mcmcconvergence}
\end{table}
 
\textcolor{black}{ 
\subsection*{Prior specification and sensitivity} \label{section:prior}
%\subsubsection{Detailed prior specification} 
To complete the Bayesian specification, here we provide details of the priors on the model parameters. In the multivariate Gaussian components of the Novelty TAGM model, as with TAGM, a common and practical choice is the use of a normal-inverse-Wishart prior. That is,
\begin{equation}
\begin{split}
\mu|&\Sigma \sim \mathcal{N}(\mu_0, \Sigma/\lambda_0) \\
&\Sigma \sim \mathcal{IW}(\nu_0, S_0) \\
& \propto \lvert\Sigma\rvert^{\frac{\nu_0 + d + 1}{2}}\exp\left[-\frac{1}{2}\text{trace}(\Sigma^{-1}S_0^{-1})\right],
\end{split}
\end{equation}
for each mixture component and where $d$ is the dimension of the data. 
To complete this discussion, we need to specify the hyperparameters, $\mu_0, \lambda_0, \nu_0$ and $S_0$. We use diffusive priors that make minimal assumptions about the data, but they are set semi-empirically as to obtain the correct scale of the data. The hyperparameters are selected as follows
\begin{equation}
\begin{split}
&\mu_0 = \frac{1}{n}\sum_{i = 1}^n{\bf x_i}, \\
&\lambda_0 = 0.01, \\
&\nu_0 = d + 2,\\
& S_0 = \frac{\text{diag }(\text{var}(X))}{K^{1/d}}.
\end{split}
\end{equation}
The hyperparameters are interpreted in the following ways. The prior mean, $\mu_0$, is the mean of the data. Then $\lambda_0$ is viewed as the number of observations with data $\mu_0$ which are added to each component specific mean. This value is small to avoid strong prior influence. The marginal prior distribution (or prior predictive) for a cluster specific mean $\mu$ is given by a student's $t$-distribution. This can be observed by recalling that the student's $t$-distribution arises by marginalisation of the covariance from a normal distribution. Now, to ensure this $t$-distribution has finite covariance we require that $\nu_0 > d + 1$. Thus, the choice presented here is the smallest integer value of $\nu_0$ that ensures a finite covariance matrix. Hence, we have a well defined $t$-distribution with heavy tails. The empirically chosen scale matrix $S_0$ is chosen to roughly partition the range of the data into $K$ balls of equal size. Previous work has shown that these priors lead to good predictive performance \citep{Crook:2018}. For $\boldsymbol{\pi}$, we take a conjugate symmetric Dirichlet prior with parameter~$\beta$, so that $\pi_1, \ldots, \pi_{K_{max}} \sim \mbox{Dirichlet}(\beta)$. Note that to apply the principle of overfitted mixtures, we have to choose $\max_j\beta_j < d/2$  \citep{Rousseau::2011}, which is satisfied in all examples by setting $\beta_j = 1$ for every $j$. Empirically \citet{Van::2015} have recommended smaller values of $\beta_j \approx n^{-1}$ to encourage stronger shrinkage.
}

\textcolor{black}{ 
\subsubsection*{Sensitivity to the choice of $\beta_j$}
To explore the sensitivity of our inferences to the specification of $\beta_j$, we considered setting $\beta_j = 0.1,0.01$, as well as $\beta_j \approx n^{-1}$ for the mESC example, which in this case $n^{-1} \approx 0.0002$. As before, we hid nucleus, chromatin and ribosome annotations and sought to use our model to rediscover them. As we now summarily describe, we found that our results can be sensitive to the choice of $\beta_j$ and hence it should be set carefully. For example when $\beta_j = 0.1$, we were unable to detect a ribosomal phenotype. Furthermore, there was a joint nucleus and chromatin phenotype, phenotype 1, rather than two distinct phenotypes.  \textit{Chromosome} was enriched for this phenotype ($p < 10^{-100}$), as well as nucleolus $(p < 10^{-60})$. When $\beta_j = 0.01$ the results were somewhat improved with a phenotype 1 enriched for \textit{chromosome} ($p < 10^{-100}$) but phenotype 3 was enriched for cytosolic ribosome $(p < 10^{-48})$ and nucleolus $(p < 10^{-50})$. Setting $\beta_j = 0.0002$ provided the expected results with 3 distinct phenotypes for \textit{chromatin} (phenotype 1) $(p < 10^{-100})$, nucleolus (phenotype 4) $(p < 10^{-50})$, and cytosolic ribsome (phenotype 3) $(p < 10^{-59})$, successfully matching our test components. Hence, based on these results, we would recommend either $\beta_j = 1$ or $\beta_j \approx n^{-1}$ depending on the desired amount of shrinkage. 
}
\textcolor{black}{ 
\subsection*{Impact of reducing the proportion of labelled proteins} \label{section:unsuper}
In all the examples we considered previously, the proportion of labelled proteins is roughly $20\%$ of the total number of proteins. %The relative proportion of labelled data might have an impact on our ability to detect novel subcellular niches, particularly in the low data setting.
  To assess the impact of the relative proportion of labelled and unlabelled proteins, we reconsidered our mESC example, where the goal was to detect ribosomal, nuclear and chromatin niches without annotation. In addition to masking these annotations as test components, we also masked, uniformly at random, an additional $10\%,20\%$ and $50\%$ of labelled proteins and assessed our ability to rediscover the ribosomal, nuclear and chromatin testing classes.
}

\textcolor{black}{ 
Briefly, we were able to rediscover two distinct phenotypes according to two nuclear clusters in all cases. When we masked $10\%$ of the labels, the enrichments for the two nuclear phenotypes were chromosome $(p < 10^{-99})$ and nucleolus $(p < 10^{-59})$, the results were the same when we removed $20\%$ and $50\%$ of labels. However, only in the scenario were $20\%$ of the labels were hidden did we find a ribosome enriched phenotype $(p < 10^{-30})$. In the other cases, the ribosome clustered with the other large protein complex: the proteasome. This reflects the similar biochemical properties of these subcellular niches. Furthermore, removing annotations renders the proteasome profile less well defined, resulting in a more diffuse cluster. In practice, careful quality control would mitigate these scenarios \citep{Gatto::2019}. In applications where there are very few annotated niches and the analysis is close to the unsupervised setting, it may be valuable to increase $K_{novelty}$ above 10 - others have found $n/2$ to work well \citep{Kirk:2012}.
}


\clearpage
\bibliographystyle{natbib}
\bibliography{BayesProt}
